src/csharp/Grpc.IntegrationTesting/ServerRunners.cs

```diff
@@ -79,8 +79,7 @@ public static IServerRunner CreateStarted(ServerConfig config)
             ServerServiceDefinition service = null;
             if (config.ServerType == ServerType.AsyncServer)
             {
-                GrpcPreconditions.CheckArgument(config.PayloadConfig == null,
-                    "ServerConfig.PayloadConfig shouldn't be set for BenchmarkService based server.");    
+                Logger.Warning("ServerConfig.PayloadConfig is not supported for C# ServerType.AsyncServer. Ignoring the value");
```

Comment:
I think the config change that added PayloadConfig to ServerConfig was https://github.com/grpc/grpc/commit/412fa2a0b7b4f84b7a6b22fccd011a69ad6f1c0f.

cc @ctiller 

I'm guessing it's ignored in other languages, as it used to not be set.